tests/TestCase/Validation/ValidationTest.php

```diff
@@ -168,6 +168,7 @@ public function testLengthBetween()
 
         $this->assertFalse(Validation::lengthBetween('abcdefg', 1, 6));
         $this->assertFalse(Validation::lengthBetween('ÆΔΩЖÇ', 1, 3));
+        $this->assertFalse(Validation::lengthBetween(1, 1, 3));
```

Comment:
I feel like this should be true actually - is there maybe a cast missing? Or are we ok with the string input only for it?